.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+name: Node CI
+
+on: [push]
```

Comment:
As a general comment I'm not highly familiar with Github Actions yet but:
- Why only on push and not on `pull_request` too?

Reply:
I don't know, i guess it's the same

Reply:
If you already have `push`, `pull_request` is only useful if you create the branch on a different repository. It doesn’t cost anything to add `pull_request` if we expect someone someday to contribute to our repository

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories-1

> When you create a pull request from a forked repository to the base repository, GitHub sends the `pull_request` event to the base repository and no pull request events occur on the forked repository.
> 
> Workflows don't run on forked repositories by default. You must enable GitHub Actions in the **Actions** tab of the forked repository.